shared/src/business/entities/cases/Case.ts

```diff
@@ -2078,7 +2076,9 @@ export const isLeadCase = rawCase =>
   rawCase.docketNumber === rawCase.leadDocketNumber;
 
 export const caseHasServedDocketEntries = rawCase => {
-  return !!rawCase.docketEntries.some(docketEntry => isServed(docketEntry));
+  return !!rawCase.docketEntries.some(docketEntry =>
```

Comment:
That is a good question... I don't know! It was here before Rachel and I touched the file, we were just attempting to refactor into using DocketEntry.isServed instead of isServed(). I will go ahead and remove it. The double bang does not seem to have an effect.